grabbit/src/main/groovy/com/twcable/grabbit/GrabbitConfiguration.groovy

```diff
@@ -151,11 +170,23 @@ class GrabbitConfiguration {
     @CompileStatic
     static class PathConfiguration {
         String path
+        Collection<String> excludePaths
         Collection<String> workflowConfigIds
 
+        void setPath(@Nullable String path) {
+            this.path = path!=null && path.endsWith("/") ? path.substring(0, path.length() - 1): path
```

Comment:
More Groovy niceness:
```groovy
path.substring(0, path.length() - 1)
```
->
```groovy
path[0..-2]
```
The negative number says "Start from the right and start going left, inclusive." Or think about it like `string.length() - n` inclusively. Since the end number of the range is inclusive, `-1` is the last character of the string.